documentation/assemblies/security/assembly-securing-access.adoc

```diff
@@ -3,48 +3,56 @@
-+
-For an added layer of protection, use the `Kafka` resource to specify authorization methods for the Kafka cluster, such as simple, OAuth 2.0, OPA, or custom authorization.
+* Listeners with specified authentication types to define how clients authenticate
+** TLS encryption for communication between Kafka and clients
+** Supported TLS versions and cipher suites for additional security
+* Authorization for the entire Kafka cluster
+* Network policies for restricting access
+* Super users for unconstrained access to brokers
 
-User accounts:: Set up user accounts and credentials with `KafkaUser` resources in Strimzi. 
-Users represent your clients and determine how they should authenticate and authorize with the Kafka cluster. 
-The authentication and authorization mechanisms specified in the user configuration must match the Kafka configuration. 
-Additionally, define Access Control Lists (ACLs) to control user access to specific topics and actions for more fine-grained authorization.
-To further enhance security, specify user quotas to limit client access to Kafka brokers based on byte rates or CPU utilization.
-+
-You can also add producer or consumer configuration to your clients if you wish to limit the TLS versions and cipher suites they use.
-The configuration on the clients must only use protocols and cipher suites that are enabled on the broker.
+Authentication is configured independently for each listener, while authorization is set up for the whole Kafka cluster.
 
-NOTE: If you are using an OAuth 2.0 to manage client access, user authentication and authorization credentials are managed through the authorization server.
+For more information on access configuration for Kafka, see the link:{BookURLConfiguring}#type-Kafka-reference[`Kafka` schema reference^] and link:{BookURLConfiguring}#type-GenericKafkaListener-reference[`GenericKafkaListener` schema reference^].
 
-Strimzi operators automate the configuration process and create the certificates required for authentication.
-The Cluster Operator automatically sets up TLS certificates for data encryption and authentication within your cluster.
+.User (client-side) configuration
 
-//Config options for securing Kafka
-include::assembly-securing-kafka-brokers.adoc[leveloffset=+1]
+To enable secure client access to Kafka, configure `KafkaUser` resources. 
+These resources represent clients and determine how they authenticate and authorize with the Kafka cluster.
+
+Configure the `KafkaUser` resource to set up the following configurations based on your specific requirements:
+
+* Authentication that must match the enabled listener authentication
+** Supported TLS versions and cipher suites that must match the Kafka configuration
+* Authorization that must match the enabled Kafka authorization
+* Access Control Lists (ACLs) for fine-grained control over user access to topics and actions
```

Comment:
The KafkaUser really supports only the simple ACL authorization. So I think we should make that clear here instead of saying it should match what is configured in the Kafka cluster.